### PR TITLE
Authentication tests

### PR DESCRIPTION
**Pull request by owencraston**
**Description**

Try to cleanup tests for the [Authentication Refactor](https://github.com/MetaMask/metamask-mobile/pull/3755) PR

**Checklist**

* [ ] There is a related GitHub issue
* [x] Tests are included if applicable
* [ ] Any added code is fully documented

**Issue**

Resolves #???

### CODE REVIEW

File: app/core/Encryptor.js [hunk 1]
```diff
@@ -19,19 +17,21 @@ export default class Encryptor {
...
+		lib === 'original'
+			? NativeModules.Aes.pbkdf2(password, salt, 5000, 256)
+			: NativeModules.AesForked.pbkdf2(password, salt);
 
 	_keyFromPassword = (password, salt, lib) => this._generateKey(password, salt, lib);
 
 	_encryptWithKey = async (text, keyBase64) => {
-		const iv = await Aes.randomKey(16);
-		return Aes.encrypt(text, keyBase64, iv).then((cipher) => ({ cipher, iv }));
+		const iv = await NativeModules.Aes.randomKey(16);
```
Comment (sethkfman) [inline app/core/Encryptor.js:27]: Remove these `Encryptor`changes

File: package.json [hunk 2]
```diff
@@ -162,6 +162,7 @@
     "path": "0.12.7",
     "pbkdf2": "3.1.2",
     "pify": "4.0.1",
+    "polyfill-crypto.getrandomvalues": "^1.0.0",
```
Comment (sethkfman) [inline package.json:165]: move this to dev dependencies

File: app/components/Views/RevealPrivateCredential/index.js [hunk 3]
```diff
@@ -29,7 +29,6 @@ import AuthenticationService from '../../../core/AuthenticationService';
 import { ThemeContext, mockTheme } from '../../../util/theme';
 import Engine from '../../../core/Engine';
 import PreventScreenshot from '../../../core/PreventScreenshot';
-import SecureKeychain from '../../../core/SecureKeychain';
```
Comment (owencraston) [inline app/components/Views/RevealPrivateCredential/index.js:32]: this was just an unused lint error

File: app/components/Views/Root/index.js [hunk 4]
```diff
@@ -34,7 +34,6 @@ export default class Root extends PureComponent {
 		if (props.foxCode === '') {
 			Logger.log('Error: foxCode is an empty string');
 		}
-		console.log('SEC INIT');
```
Comment (owencraston) [inline app/components/Views/Root/index.js:37]: This was causing a lint error

File: app/components/Views/ImportFromSeed/index.js [hunk 5]
```diff
@@ -19,7 +19,6 @@ import { connect } from 'react-redux';
 import { passwordSet, seedphraseBackedUp } from '../../../actions/user';
 import { setLockTime } from '../../../actions/settings';
 import StyledButton from '../../UI/StyledButton';
-import Engine from '../../../core/Engine';
```
Comment (owencraston) [inline app/components/Views/ImportFromSeed/index.js:22]: this was just an unused import causing a lint error

**Review by sethkfman — APPROVED** [review]
LGMT

**Pull request merged**